src/screens/Events/EventsMap.js

```diff
@@ -81,13 +85,13 @@ export default class EventsMap extends Component {
     }, 1500);
   }
 
-
-  async getEvents(newEvent) {
+  populateMap(newEvent) {
     try {
-      const response = await eventServices.gets();
-      if (response instanceof Error) throw response;
-      this.setState({ events: response.data }, () => {
-        const { events } = this.state;
+      // if (response instanceof Error) throw response;
+      // this.setState({ events: response.data }, () => {
+      //   const { events } = this.state;
+      // });
+      this.props.events.forEach(event => {
         if (newEvent) this.focusMarker(newEvent);
```

Comment:
Why do we have to loop through all events in order to determine whether we need to focus on `newEvent`? It seems like this function is supposed to be putting all events on the map, but it doesn't look like that's what it's doing. Am I missing something?